geometry_msgs/Pose pose
float32[] joint_values
float32 vel_scale
float32 acc_scale
string planner # Allowed values: "ompl", "ptp", "lin"